documentation/modules/exploit/windows/http/ektron_xslt_exec_ws.md

```diff
@@ -0,0 +1,70 @@
+## Vulnerable Application
+
+  Ektron Content Management System (CMS) 8.5 and 8.7 before 8.7sp2 and 9.0 before sp1 (according to CVE-2015-0923)
```

Comment:
Ektron, when they were Ektron, patched this. Epi Server should *not* be vulnerable.

I too, do not understand their version numbers. I believe 8.0, 8.5, 9.0 all represent separate releases and the service pack match between these  versions. So 9.0 sp1 for instance would correspond to 8.5 sp2. I suspect this ties in strongly with database structure and the like, but I'm not sure.

I do not know where you can find installers.